triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -693,11 +712,6 @@ private[lf] class Runner private (
         )
     }
 
-    val logInitialState =
```

Comment:
In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity